analysis/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Wed Sep 23 13:22:07 CEST 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-all.zip
```

Comment:
I suggest to use a variable for the version here to keep the version up to date.